internal/comm/httptransport.go

```diff
@@ -63,7 +63,7 @@ func NewHTTPTransport(config *Config) *HTTPTransport {
 		logger:         config.Logger,
 		localConf:      config.LocalConf,
 		catchUpClient:  NewCatchUpClient(config.Logger),
-		catchupHandler: NewCatchupHandler(config.Logger, config.LedgerReader),
+		catchupHandler: NewCatchupHandler(config.Logger, config.LedgerReader, 0), //TODO make max-response-bytes configurable
```

Comment:
I assume it to be part of the local config file. I am also fine if we fix it to 100 MB always and keep the config file with a smaller number of important parameters.